addons/web_editor/static/src/js/wysiwyg/widgets/link_dialog.js

```diff
@@ -12,7 +13,12 @@ export class LinkDialog extends Link {
         focusField: { type: String, optional: true },
         close: { type: Function },
         onSave: { type: Function },
+        hasButtonDialog: { type:Boolean, optional: true},
     };
+    static defaultProps = {
+        title: _t("Insert a Button"),
+        confirmText: _t("Insert"),
+    }
```

Comment:
Would not add props for the title and confirm text, it complexifies the Component.

We can just hardcode "Insert a Link / Button" for the title and "Insert" for the button.